packages/styles/src/core/variables/_zindex.scss

```diff
@@ -0,0 +1,10 @@
+$z-index-0: 0;
```

Comment:
😍

Reply:
Actually, it'd be better with meaningful names, but we can do that in the future ✨